web/src/components/network/NetworkPage.jsx

```diff
@@ -177,15 +177,40 @@ export default function NetworkPage() {
 
   return (
     <>
-      <Section title={_("Wired connections")} icon="lan">
-        {ready ? <WiredConnections /> : <Skeleton />}
-      </Section>
-
-      <Section title={_("WiFi connections")} icon="wifi">
-        {ready ? <WifiConnections /> : <Skeleton />}
-      </Section>
-
-      <NetworkPageMenu wifiScanSupported={wifiScanSupported} openWifiSelector={openWifiSelector} />
+      <Page.Header>
+        <Flex>
+          <FlexItem>
+            <h2>{_("Network")}</h2>
+          </FlexItem>
+          <If
```

Comment:
Note, I'm killing the `<If>` component too. There were no few complains about it in the past :)

Reply:
Also, another note: for the shake of reducing nodes, I'm using Flex with props when possible instead of Flex with FlexItems. But this is just a comment, please keep the change as it is.